llpy/agency.py

```diff
@@ -0,0 +1,12 @@
+from llpy import LLpyException, Base
+from llpy.constants import URL
+
+
+class Agency(Base):
+    instance_name = ['agency', 'agencies']
+    def __get_the(self, query):
+        return Base.__get_the(self, query)
```

Comment:
you can't to this as far as I know :/

Reply:
test case:
```
class A:
   field = 'aaaa'
   def pr(self):
       print(self.field)

class B(A):
   field = 'bbbb'

def main():
   a = A()
   b = B()
   a.pr()
   b.pr()
```

if you don't mean initing 'instance_name', then I didn't get the thing. :)

Reply:
I mean, getting private methods from another class with this syntax.